src/main/java/org/fcrepo/migration/validator/impl/ApplicationConfigurationHelper.java

```diff
@@ -95,6 +111,47 @@ private ObjectSource doObjectSource () throws IOException {
         return objectSource;
     }
 
+
+    private MutableOcflRepository repository(final Fedora3ValidationConfig config, final Path workDir) {
```

Comment:
It is likely safer if we create an "Immutable" OCFL repository... since we are just performing validation.

Reply:
The fcrepo-storage-ocfl's DefaultDefaultObjectSessionFactory doesn't currently have a constructor with OcflRepository.  I will create a ticket to for this.

Reply:
https://jira.lyrasis.org/browse/FCREPO-3631

Reply:
and https://jira.lyrasis.org/browse/FCREPO-3632